myokit/formats/sbml/_importer.py

```diff
@@ -67,21 +66,34 @@ def _convert_unit(self, unit):
     def info(self):
         return info
 
-    def model(self, path):
+    def model(self, path, bind_time=True):
+        """
+        Returns a :class:myokit.Model based on the SBML file provided.
 
+        Arguments:
+            path -- Path to SBML file.
```

Comment:
Please see e.g. [https://github.com/MichaelClerx/myokit/blob/master/myokit/_model_api.py#L319](here) for the syntax we're using in Myokit.